.github/workflows/code-nightly.yaml

```diff
@@ -34,8 +34,13 @@ jobs:
           headCommit=$(curl -H 'Accept: application/vnd.github.VERSION.sha' https://api.github.com/repos/gitpod-io/openvscode-server/commits/gp-code/main)
           cd components/ide/code
           leeway build -Dversion=nightly -DimageRepoBase=eu.gcr.io/gitpod-core-dev/build -DcodeCommit=$headCommit -DcodeQuality=insider .:docker
+      - name: Get previous job's status
+        id: lastrun
+        uses: filiptronicek/get-last-job-status@main
+        with:
+          workflow-id: 15751907
```

Comment:
using the number id this looks tricky, it think it would be better to use the filename

Reply:
I would have guessed the workflow id would appear in the GHA's URL but it doesn't. Where do you get it from?

For me it's non-blocking but I agree it's more friendly to use the filename (although I suspect the workflow ID will stay the same even if the filename changes)

Reply:
Thanks @jeanp413, I totally agree the file name should be used instead. The reason I chose the workflow ID is because that's what the API call wants, so I will need to first query what the ID of the corresponding file name is. Should be easy though, will work on it.

Reply:
you can use the filename directly https://octokit.github.io/rest.js/v18#actions-list-workflow-runs

Reply:
Missed this: 🤦‍♂️
> You can also pass the workflow file name as a string.

Reply:
Should now work without specifying anything.